amazon-arm64.pkr.hcl

```diff
@@ -0,0 +1,205 @@
+variable "ami" {
+  type    = string
+  default = "ubuntu/images/hvm-ssd/ubuntu-focal-20.04-arm64-server-*"
+}
+
+variable "ami_name" {
+  type    = string
+  default = "supabase-postgres-arm-13.3.0.4"
```

Comment:
let's remove this default - fail if not provided

Reply:
ok, updated the PR